### PR TITLE
Add Book feature

### PR DESCRIPTION
**Pull request by Aqurds**
The following features were added,
 
- Setup a new React application using [Create React App](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app)
- Created a directory for your reusable components: /components.
- Installed [React Router V6](https://reactrouter.com/en/main/start/tutorial#setup)
- Add 2 routes in your application, each should render a page component:
    - /
    - /categories
- Created components inside of the /components directory:
- Contains individual book states, as well as a button to delete the book
- Contains a list to render individual books
- Contains a form to create new books
- Contains navigation to link to the 2 routes you created

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,7 +144,7 @@ Clone this repository to your desired folder:
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](https://github.com/Aqurds/ds-algo-track/issues).
+Feel free to check the [issues page](https://github.com/Aqurds/bookstore/issues).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (Owusu-Desmond) [inline README.md:149]: - [x] Please make sure when the `MIT` link is clicked it takes you to the MIT license page of your project. I can see dont have a license file or your license link is broken. You can add a license file by clicking on the `Add a license` button on the right side of your project page. You can read more about it [here](https://docs.github.com/en/github/building-a-strong-community/adding-a-license-to-a-repository) :pray:

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @Aqurds , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Aqurds) [reply README.md:149]: License added

**Review by shella12 — APPROVED** [review]
Hi @Aqurds, 


## Status : APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Added LICENCE ✔️ 
- README is professional 👍 
- Implemented all requirements 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**